yang/IETF-04/ietf-vpn-common.yang

```diff
@@ -429,29 +429,35 @@ module ietf-vpn-common {
        where Hubs can act as Spoke for certain part of
        the network or Spokes as Hubs.";
   }
-  
+    
   identity role {
-   description
-     "Base identity for site or node role type.";
-  }
-  
-  identity any-to-any-role {
-   base role;
-   description
-     "Site/Node in an any-to-any VPN.";
-  }
-  
-  identity spoke-role {
+    description
+    "Base identity for site type.";
+   }
+
+   identity any-to-any-role {
     base role;
     description
-      "Spoke site/node in a Hub-and-Spoke IP VPN.";
-  }
-  
-  identity hub-role {
+    "Site in an any-to-any IP VPN.";
+   }
+
+   identity spoke-role {
     base role;
     description
-      "Hub site/node in a Hub-and-Spoke VPN.";
-  }
+    "Spoke site in a Hub-and-Spoke IP VPN.";
+   }
+
+   identity hub-role {
+    base role;
+    description
+    "Hub site in a Hub-and-Spoke IP VPN.";
```

Comment:
Please mention "node" as we are also using it for things such as:

              +--rw node-role?                 identityref

Reply:
ae785b919c19f6a03564c9529fa39b0495983d48